src/main/java/io/micronaut/build/testresources/TestResourcesHelper.java

```diff
@@ -115,6 +123,13 @@ public void start() throws MojoExecutionException {
     }
 
     private void doStart() throws DependencyResolutionException, IOException {
+        if (shared) {
+            if (sharedServerNamespace != null) {
+                log.info("Test Resources is configured in shared mode with the namespace: " + sharedServerNamespace);
+            } else {
+                log.info("Test Resources is configured in shared mode");
+            }
+        }
```

Comment:
@melix is this logic in the Gradle plugin as well?  So namespace requires shared to be enabled?

Reply:
That's correct, namespaces are for shared only.